tests/end-to-end/external-solution-integration/internal/scenario/send_and_check_event_only/send_and_check_event_only.go

```diff
@@ -41,7 +36,7 @@ func (s *SendEventAndCheckCounter) Steps(config *rest.Config) ([]step.Step, erro
 
 	return []step.Step{
 		testsuite.NewConnectApplication(connector, state, s.ApplicationTenant, s.ApplicationGroup),
-		testsuite.NewSendEvent(s.testID, helpers.LambdaPayload, state),
+		testsuite.NewSendEventToMesh(s.testID, helpers.LambdaPayload, state),
```

Comment:
`NewSendEventToMesh` is one scenario where cloud events are sent to `/events`, which is good. But the most used scenario is to send events to `"https://gateway.%s/%s/v1/events` which is happening in `NewSendEvent` and this will make the event go through the compatibility layer and finally to the event-mesh. We may only test this, IMO cause the `NewSendEventToMesh` is a subset of `NewSendEvent`

Reply:
Good catch

Reply:
OTOH v1 is just the compatibility Layer... and it should be removed soon

Reply:
i will introduce a sendevent method that sends to v1 ... this should then also cover @anishj0shi's concerns

Reply:
fixed